packages/authentication/package.json

```diff
@@ -4,9 +4,10 @@
   "description": "A LoopBack component for authentication support.",
   "scripts": {
     "acceptance": "mocha --opts ../../test/mocha.opts 'test/acceptance/**/*.ts'",
-    "build": "npm run build:lib && npm run build:lib6",
+    "build": "npm run build:lib && npm run build:lib6 && npm run build:apidocs",
```

Comment:
AFAIK, no other LoopBack packages are running `sdocs` as part of pre-publishing. The code powering http://apidocs.strongloop.com (re)builds API doc on demand. This has one important benefit - we can render API docs using the latest strong-docs version, even a strong-docs version that's newer than the published package.

So, why are we adding this step here in loopback-next?